config/default.yml

```diff
@@ -12,16 +12,43 @@ Lint/NestedMethodDefinition:
   Exclude:
     - api/sinatra/**/*
 
+Lint/RaiseException:
+  Enabled: true
+
+Lint/StructNewOverride:
+  Enabled: true
+
 ### Metrics
 
-Layout/LineLength:
+Metrics/AbcSize:
+  Max: 18
+  Severity: refactor
+  Exclude:
+    - api/sinatra/**/*
+
+Metrics/BlockLength:
+  Exclude:
+    - '*.gemspec'
+    - '**/*.rake'
+    - 'config/initialize/*'
+    - 'spec/**/*'
+    - 'api/**/*'
```

Comment:
this is not fitting the shed structure

Reply:
@ThomasKoppensteiner please update with the relevant shed structures